manager/integration/tests/common.py

```diff
@@ -3279,18 +3294,24 @@ def wait_for_all_instance_manager_running(client):
 
     for i in range(RETRY_COUNTS):
         instance_managers = client.list_instance_manager()
-        node_to_engine_manager_map, node_to_replica_manager_map = {}, {}
+        node_to_engine_manager_map = {}
+        node_to_replica_manager_map = {}
+        node_to_instance_manager_map = {}
         try:
             for im in instance_managers:
                 if im.managerType == "engine" and im.currentState == "running":
                     node_to_engine_manager_map[im.nodeID] = im
                 elif im.managerType == "replica" and \
                         im.currentState == "running":
                     node_to_replica_manager_map[im.nodeID] = im
+                elif im.managerType == "aio" and \
+                        im.currentState == "running":
+                    node_to_instance_manager_map[im.nodeID] = im
                 else:
                     print("\nFound unknown instance manager:", im)
             if len(node_to_engine_manager_map) != len(nodes) or \
-                    len(node_to_replica_manager_map) != len(nodes):
+                    len(node_to_replica_manager_map) != len(nodes) or \
+                    len(node_to_instance_manager_map) != len(nodes):
```

Comment:
In fresh install, the number of engine/replica instance manager should be 0.
In upgrade, the number of existing engine/replica should depend on how many attached volumes are still running. 

Why we expect the number of each instance manager type should be equal to the node number?

Reply:
Good question. After checking the code again, the `wait_for_all_instance_manager_running` is used in `cleanup_client` for the client fixture. Longhorn will get upgraded during the `test_upgrade` and cleaning up shouldn't require checking on the instance-manager-e/r.

Removed.